treeherder/config/settings.py

```diff
@@ -310,7 +318,7 @@
 CELERY_TASK_CREATE_MISSING_QUEUES = False
 
 # Celery broker setup
-CELERY_BROKER_URL = env('BROKER_URL')
+CELERY_BROKER_URL = env('BROKER_URL', default='localhost//guest:guest@rabbitmq//')
```

Comment:
On PR #5943 I discovered that the correct value is `amqp://guest:guest@localhost:5672//`